.github/actions/setup-binary/action.yml

```diff
@@ -150,3 +153,13 @@ runs:
         # Add binary to path
         echo "Adding ${INSTALL_PATH} to PATH"
         echo "${INSTALL_PATH}" >> $GITHUB_PATH
+
+    - name: 'Mark binary as executable'
+      if: |
+        inputs.mark_executable != ''
+      shell: 'bash'
+      env:
+        INSTALL_PATH: '${{ inputs.install_path }}'
+        MARK_EXECUTABLE: '${{ inputs.mark_executable }}'
+      run: |-
+        chmod +x ${INSTALL_PATH}/${MARK_EXECUTABLE}
```

Comment:
The binary could be nested in the install path, the name mark_executable is kind of confusing. Maybe we separate this in to two inputs

`set_executable` - default false
`binary_subpath` - only used if set_executable = true

See https://github.com/abcxyz/pkg/blob/5c6ce400b0f1f7fc315e0cc5a8e9f6699fe6ef16/.github/actions/setup-binary/action.yml#L145 on testing for truthy values

Reply:
Separated into two inputs.